conf/susi/susi_cognition_000.json

```diff
@@ -403,6 +404,268 @@
 			"actions":[	{"type":"answer", "select":"random", "phrases":[
 							"You are welcome, too."
 						]}]
+		},
+		{
+
```

Comment:
Unwanted space, Please remove these spaces we don't want the file to get bigger with all these where it can be avoided